install/pre-install-payload/scripts/reqs-deploy.sh

```diff
@@ -167,8 +167,13 @@ function uninstall_artifacts() {
 
 function restart_systemd_service() {
 	host_systemctl daemon-reload
-	echo "Restarting ${container_engine}"
-	host_systemctl restart "${container_engine}"
+	if [ "${container_engine}" == "cri-o" ]; then
+		service_name="crio"
+	else
+		service_name="${container_engine}"
+	fi
+	echo "Restarting ${service_name}"
+	host_systemctl restart "${service_name}"
```

Comment:
I'd rather solve this here: https://github.com/confidential-containers/operator/pull/424/files#diff-1d64c80e2d9aea0884eef362ad623406c0edb4de1295dd31579a9270bc428c3eR44

This is what we do on the Kata Containers side: https://github.com/kata-containers/kata-containers/blob/ef5a5ea26e893b1f7fa20bd82eca402b11f08b21/tools/packaging/kata-deploy/scripts/kata-deploy.sh#L609-L610